<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>424</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Copy or Move File (Demo)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="move">
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Src: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sourceFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enter Source File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="selectFile">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Dst:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="targetFolder">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enter Destination Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="selectFolder">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="changeName">
      <property name="title">
       <string>Change Name</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="newName"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>171</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="title">
       <string>Invalidate</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QCheckBox" name="invalidateSrc">
           <property name="text">
            <string>Source</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="invalidateDst">
           <property name="text">
            <string>Destination</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="goButton">
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>433</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>goButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onGo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSelectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectFolder</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSelectFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>validate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changeName</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>validate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onGo()</slot>
  <slot>onSelectFile()</slot>
  <slot>onSelectFolder()</slot>
  <slot>validate()</slot>
 </slots>
</ui>
